### ISSUE TITLE
Incorrect rounding?

### ISSUE THREAD

**Issue by RachelScodes** [MikeMcl/bignumber.js#242]
Not sure if my expectations are wrong, or this is a bug.

I am passing in a default rounding value in my config like so:
`BigNumber.config({ ROUNDING_MODE: BigNumber.<UP/CEIL/HALF> })`

Given two variables (not static values, they derive from addition/multiplication from user input):
```
a = 0.6432
b = 0.2447
```

If I use ROUND_CEIL, ROUND_UP or ROUND_HALF_UP I get the following:
❌ `a.toFixed(2) = '0.65'` I want .64, not .65
✅ `b.toFixed(2) = '0.25'`

However, if I use ROUND_HALF_CEIL I get the following:
✅ `a.toFixed(2) = '0.64'` 
❌ `b.toFixed(2) = '0.24'` I want .25, not .24

What do I need to do to get:
✅ a.toFixed(2) = '0.64' 
✅ b.toFixed(2) = '0.25'

?

EDIT: to be very clear - I do not know what these values will be ahead of time, but I want to round them correctly.

**Comment by MikeMcl** [comment]
The rounding modes are described [here](http://mikemcl.github.io/bignumber.js/#constructor-properties).

```js
BigNumber.config({ ROUNDING_MODE: BigNumber.ROUND_HALF_UP });
new BigNumber(0.6432).toFixed(2);    // "0.64"

BigNumber.config({ ROUNDING_MODE: BigNumber.ROUND_UP });
new BigNumber(0.2447).toFixed(2);    // "0.25"

// Alternatively
new BigNumber(0.2447).toFixed(2, BigNumber.ROUND_CEIL );    // "0.25"
```

Quel est le problème ?

**Comment by RachelScodes** [comment]
again, I do not know what these values will be ahead of time. I can't set the rounding option in the config to be different for each number if i don't know what the numbers will be.

**Comment by MikeMcl** [comment]
Far from being"very clear", you have not given the criteria which determines how you want a value to be rounded.

Anyway, I recommend you ask the question on [stackoverflow.com](https://stackoverflow.com/).

**Comment by RachelScodes** [comment]
1. "you have not given the criteria which determines how you want a value to be rounded."

What? That's the only information in the ticket: the numbers, how they should be rounded, and asking what config value does that for **both** cases. You didn't answer the question, so I rephrased it.

2. This isn't a general how-to question, it's about this particular library which you maintain. If you think someone random on the internet would be better equipped to answer this question than you - the owner of the repo - then I'm a little disappointed is all.

Closing a question because you don't know the answer doesn't mean it's resolved. Thanks for your time.
